ui/src/containers/AppDetailsContainer.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import AppsTable from '../components/AppsTable';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { Checkbox } from '@patternfly/react-core';
+import { getApps, reverseAppDetailsTableSort } from '../actions/actions-ui';
+
+export class AppDetailsContainer extends React.Component {
+  constructor (props) {
+    super(props);
+    this.onSort = this.onSort.bind(this);
+    this.onRowClick = this.onRowClick.bind(this);
+    this.getTable = this.getTable.bind(this);
+  }
+
+  componentWillMount () {
```

Comment:
We'll want to use a getApp here. I believe @craicoverflow is working on getting the app fro the backend.

Reply:
agree - we can include that in the PR that adds the API get call

Reply:
resolving as should be included in AEROGEAR-8466